app/com/linkedin/drelephant/spark/fetchers/SparkRestClient.scala

```diff
@@ -89,6 +93,36 @@ class SparkRestClient(sparkConf: SparkConf) {
     }
   }
 
+  def fetchLogData(appId: String, attemptId: Option[String])(
+      implicit ec: ExecutionContext
+  ): Future[Option[SparkLogDerivedData]] = {
+    val appTarget = apiTarget.path(s"applications/${appId}")
+    logger.info(s"calling REST API at ${appTarget.getUri}")
+
+    val logStem = attemptId.map(id => s"${appId}_$id").getOrElse(appId)
```

Comment:
I've decided to remove it completely, it's not required, because Spark guarantees the archive would only have a single entry.